samples/Sentry.Samples.Maui/MainPage.xaml.cs

```diff
@@ -27,6 +29,28 @@ protected override void OnAppearing()
         base.OnAppearing();
     }
 
+    private void AddHandlerBreadcrumbs()
+    {
+        foreach (var visualElement in this.GetVisualTreeDescendants())
```

Comment:
nit: it's by default on `this` already

```suggestion
        foreach (var visualElement in GetVisualTreeDescendants())
```